src/graphql/data/RelayEnvironment.ts

```diff
@@ -1,9 +1,37 @@
-import { Environment, Network, RecordSource, Store } from 'relay-runtime'
+import ms from 'ms.macro'
+import { Variables } from 'react-relay'
+import { Environment, Network, RecordSource, RequestParameters, Store } from 'relay-runtime'
+import RelayQueryResponseCache from 'relay-runtime/lib/network/RelayQueryResponseCache'
 
 import fetchGraphQL from './fetchGraphQL'
 
+export const cache = new RelayQueryResponseCache({ size: 250, ttl: ms`5m` })
```

Comment:
did you figure out why 250 here?

Reply:
Yes, 250 means "max number of request in cache, least-recently updated entries purged first (default: 100)" from https://github.com/relay-tools/react-relay-network-modern. I'll add a comment explaining

Reply:
I'm not sure what the best number to choose is here, 250 was just in their sample. Very open to suggestions

Reply:
idk i'd use the default of 100 then if you don't have a clear reason why 250